src/app/services/views/ServiceDetailTabsView.vue

```diff
@@ -52,8 +52,19 @@
 
         <template v-else>
           <NavTabs
-            class="route-service-detail-view-tabs"
-            :tabs="getNavTabs(data)"
+            anchor-route-name="service-detail-tabs-view"
+            i18n-prefix="services.routes.item.navigation"
+            :filter-predicate="(route) => {
+              if (data.serviceType !== 'external' && route.name === 'service-config-view') {
+                return false
+              }
+
+              if (data.serviceType === 'external' && route.name === 'service-data-plane-proxies-view') {
+                return false
+              }
+
+              return true
+            }"
```

Comment:
**Comment**: This won’t be necessary once we remove the KUMA_GATEWAYS_UI feature flag and can be removed again.